ACS/Source/AssetCatalogReader.m

```diff
@@ -259,7 +259,7 @@ - (void)readThemeStoreWithCompletionHandler:(void (^__nonnull)(void))callback pr
                 return;
             }
                 
-            const BOOL isSVG = rendition.isVectorBased && rendition.svgDocument;
+            const BOOL isSVG = rendition.isVectorBased && rendition.svgDocument && CGSVGDocumentGetCanvasSize && CGContextDrawSVGDocument && CGSVGDocumentWriteToData;
```

Comment:
AFAIK you have to explicitly compare against `NULL` in order to avoid a crash if the symbols are missing.

Reply:
surprising, but should be done